profiling/src/profile/mod.rs

```diff
@@ -433,6 +438,12 @@ impl Profile {
             .insert(interned_span_id, interned_endpoint);
     }
 
+    pub fn add_endpoint_count(&mut self, endpoint: Cow<str>, value: i64) {
+        self.endpoints
+            .stats
+            .add_endpoint_count(endpoint.to_string(), value);
```

Comment:
Here you want [into_owned](https://doc.rust-lang.org/std/borrow/enum.Cow.html#method.into_owned) I think.
To owned will no copy the data if the Cow is already of the owned variant, whereas to_string calls the fmt::Display traits which always allocates and clones the data.